pisa-proxy/parser/mysql/src/grammar.y

```diff
@@ -2372,7 +2375,7 @@ opt_expr_list -> Vec<Expr>:
   ;
 
 expr_list -> Vec<Expr>:
-    expr
+    expr %prec LOWER_THEN_COMMA
```

Comment:
Is there conflict here?

Reply:
Ok. Currently `Shift(",") / Reduce(expr_list: "expr")`  conflict  has not reloved yet.